.github/workflows/tests.yml

```diff
@@ -1,28 +1,25 @@
 name: Tests
 
+# Runs every day at 06:00 UTC
```

Comment:
Why are you overwriting `tests.yml` instead of creating a new workflow file?

Reply:
I was testing out something I saw on an issue saying the workflow must already exist for you to be able to run it before merging it (it did not work). Removing the commit.